docs/index.md

```diff
@@ -0,0 +1,4 @@
+# Documentation
```

Comment:
Could you write intro for ostis-example-app?

For example, see:

https://ostis-ai.github.io/sc-machine/
https://ostis-ai.github.io/scp-machine/